2008-04-17  Arnaud Charlet  <charlet@adacore.com>

	* pixman/pixman.h: Fix compilation failure on solaris
	Submitted at https://bugs.freedesktop.org/show_bug.cgi?id=15560

--- pixman/pixman.h.old	2008-04-17 11:31:07.702107817 +0200
+++ pixman/pixman.h	2008-04-17 11:31:44.522472531 +0200
@@ -76,7 +76,6 @@ SOFTWARE.
  */
 #if defined (__SVR4) && defined (__sun)
 #  include <sys/int_types.h>
-#  include <stdint.h>
 #elif defined (__OpenBSD__)
 #  include <inttypes.h>
 #elif defined (_MSC_VER)
